IdentityCore/src/broker_operation/response/MSIDDeviceInfo.m

```diff
@@ -63,7 +76,7 @@ - (instancetype)initWithJSONDictionary:(NSDictionary *)json error:(__unused NSEr
         
         NSData *jsonData = [json[MSID_ADDITIONAL_EXTENSION_DATA_KEY] dataUsingEncoding:NSUTF8StringEncoding];
         _additionalExtensionData = [NSJSONSerialization msidNormalizedDictionaryFromJsonData:jsonData error:nil];
-        _mdmId = [json msidStringObjectForKey:MSID_BROKER_MDM_ID_KEY];
+        _extraDeviceInfo = [json msidObjectForKey:MSID_EXTRA_DEVICE_INFO_KEY ofClass:[NSDictionary class]];
```

Comment:
Looks like you're initializing this with a non-mutable dictionary

Reply:
I think it is ok to assign a reference to a non-mutable dictionary, as long as we don't change anything inside the dictionary

Reply:
changed to non mutable dict